Extension/src/LanguageServer/configurations.ts

```diff
@@ -1241,6 +1238,17 @@ export class CppProperties {
             this.resetToDefaultSettings(true);  // I don't think there's a case where this will be hit anymore.
         }
 
+        const configuration: Configuration | undefined = this.CurrentConfiguration;
+        if (configuration) {
+            if (configuration.configurationProvider !== undefined) {
+                getUI().then((ui: LanguageStatusUI) => ui.ShowConfigureIntelliSenseButton(false, this.client, ConfigurationType.ConfigProvider, "baseConfiguration"));
+            } else if (configuration.compileCommands !== undefined) {
+                getUI().then((ui: LanguageStatusUI) => ui.ShowConfigureIntelliSenseButton(false, this.client, ConfigurationType.CompileCommands, "baseConfiguration"));
+            } else if (configuration.compilerPath !== undefined) {
```

Comment:
This is logging compilerPath == "" as ConfigurationType.CompilerPath instead of ConfigurationType.NotConfigured.

Reply:
You're right.  Setting compilerPath to "" should count as "configured."

Reply:
Oh, is that configured, I'm not sure, I just noticed it was inconsistent.

Reply:
I don't think we show the "configure IntelliSense" button if they set compilerPath to `""`, do we?